flamingo/plugins/feeds.py

```diff
@@ -46,10 +66,24 @@ def pre_build(self, context):
 
                     if 'updated' in feed_config:
                         fe_updated = feed_config['updated'](i)
-
                     else:
                         fe_updated = ''
 
+                    if 'published' in feed_config:
+                        fe_published = feed_config['published'](i)
+                    else:
+                        fe_published = ''
+
+                    if feed_config['type'] == 'podcast':
+                        if 'podcast' in i:
+                            fe_podcast_url = i['podcast'].get('url', '')
+                            fe_podcast_size = i['podcast'].get('size', 0)
+                            fe_podcast_type = i['podcast'].get('type', 'audio/mpeg')
+                        else:
+                            fe_podcast_url = ''
+                            fe_podcast_size = ''
+                            fe_podcast_type = 'audio/mp3'
```

Comment:
> The value assigned to `fe_podcast_type` is just used as a placeholder to signal that it has a valid default and never has to be added to `missing_attributes`.
> This took me a moment to comprehend but sounds reasonable.

The structure in this file is not that straight forward. Mixing a 3rd kind of feed in didn't do readability any good I guess.



> This took me a moment to comprehend but sounds reasonable.
> I do however think the mime type here and above should be the same, not `audio/mp3` and `audio/mpeg`.

Totally right. Fixed.


> Also the check for `feed_config['type'] == 'podcast'` is not strictly necessary, is it? The values will just be ignored later if it is not.

Removed that check.


> My go to way of writing this would look something like this, but you solution better matches the surrounding code, so it is preferable:

Ack, totally my thoughts. :-)